### ISSUE TITLE
Plugin throwing error on Vim 8

### ISSUE THREAD

**Issue by hrai** [Ben201310/online-thesaurus-vim#2]
I get this error when I run the plugin command.

I'm running vim-gtk on Ubuntu 18.x with Python compatibility and Python installed.

**Comment by Ben201310** [comment]
Hi, 

Unfortunately I don't have a ubuntu 18.x installed on my machine, but I have tried the plugin on Windows 10, Mac OS High Serria and Ubuntu 16.04 and they all seem to work fine. The only thing I can think of at this moment is whether the supported python version in your vim is compatible with your python distribution installed (see the FAQ for more information). If you find they are indeed compatible, you are welcome to submit a minimal working example (and without other plugins idealy), I will have a look at it when I find a machine having Ubuntu 18.x installed.

**Comment by hrai** [comment]
This only worked on vim with python2 support.

I had to install `vim-gtk`

**Comment by Ben201310** [comment]
Good point! I haven't tried Vim with only python 3 support yet. I will have to have a look at this as soon as possible and report findings after that.

**Comment by Ben201310** [comment]
Hello. Sorry it has been a while - I promised that I would have a look at this issue and I just managed to do that. The issue seemed to arise from the "python ..." statement which does not switch to python3 if no python2 is supported. I have made changes in the file.  Now python2 support is assumed by default and if only python3 is supported this can be switch on by putting "set pyx=3" in .vimrc. If this solved you problem I'd close this issue. Cheers.
